video/out/w32_common.c

```diff
@@ -1055,6 +1055,32 @@ static void update_dark_mode(const struct vo_w32_state *w32)
     DwmSetWindowAttribute(w32->window, DWMWA_USE_IMMERSIVE_DARK_MODE,
                           &use_dark_mode, sizeof(use_dark_mode));
 }
+#if __MINGW64_VERSION_MAJOR >= 12
+enum DWM_SYSTEMBACKDROP_TYPE intToBackdrop(int value) {
+    switch (value) {
+        case 1:
+            return DWMSBT_NONE;
+        case 2:
+            return DWMSBT_MAINWINDOW;
+        case 3:
+            return DWMSBT_TRANSIENTWINDOW;
+        case 4:
+            return DWMSBT_TABBEDWINDOW;
+        default:
+            return DWMSBT_AUTO;
+    }
+}
+
+static void update_backdrop(const struct vo_w32_state *w32, bool init)
+{
+    if (w32->parent || (w32->opts->backdrop_type == 0 && init))
```

Comment:
Why `w32->opts->backdrop_type == 0`? `0` is mapped to `DWMSBT_AUTO`, so it is fine to set it as default value.